project/AppDependencies.scala

```diff
@@ -17,6 +17,7 @@ object AppDependencies {
     "org.jsoup" % "jsoup" % "1.16.1" % Test,
     "com.vladsch.flexmark" % "flexmark-all" % "0.35.10" % Test,
     "org.scalatestplus.play" %% "scalatestplus-play" % "5.1.0" % Test,
-    "org.mockito" %% "mockito-scala-scalatest" % "1.17.14" % Test
+    "org.mockito" %% "mockito-scala-scalatest" % "1.17.14" % Test,
+    "uk.gov.hmrc" %% "play-frontend-hmrc" % "7.23.0-play-28" % Test
```

Comment:
I think this play-frontend-hmrc % Test is redundant, what happens if you remove it?

Reply:
Was just following the convention as in the above dependencies but sure will remove it